core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -101,6 +101,10 @@
       <div ng-show="getTabStatuses().active === 'feedback'">
         {% include 'pages/exploration_editor/feedback_tab/feedback_tab.html' %}
       </div>
+
+      <div ng-show="getTabStatuses().active === 'issues'">
+        {% include 'pages/exploration_editor/issues_tab/issues_tab.html' %}
```

Comment:
Could we reference the issues tab directive directly rather than introducing another Jinja dependency?

@vojtechjelinek are you aware of any reason why this can't be a directive?

Reply:
Looks like he was following the pattern used to display other tabs. If we do it, would we like to do it all together? So that the code used for loading a tab is consistent.

Reply:
Hmm, I'd rather do such a migration in a separate PR tbh.

Reply:
Since #3950 is tracking progress for this overall effort, I'll introduce the improvements tab as a directive in this PR and leave the others for that issue.